codegen/smithy-dafny-codegen/src/main/java/software/amazon/smithy/dafny/codegen/DafnyClientCodegenPlugin.java

```diff
@@ -35,7 +35,7 @@ public void execute(PluginContext context) {
                 .orElseThrow(() -> new RuntimeException("Invalid plugin settings; aborting"));
 
         final Map<TargetLanguage, Path> outputDirs = new HashMap<>();
-        outputDirs.put(TargetLanguage.DAFNY, manifest.resolvePath(Paths.get("Model")));
+        //outputDirs.put(TargetLanguage.DAFNY, manifest.resolvePath(Paths.get("Model")));
```

Comment:
Why are commenting this out?

Reply:
This was for Tests, now that I've a working TestModel project, added this back.